notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -179,7 +179,7 @@
         "! (pip3 install --upgrade $USER_FLAG \\\n",
```

Comment:
<p>TIMESTAMP -&gt; UUID</p><p>Remove trailing commas: q_source=[TRAINING_DATASET_BQ_PATH],</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/985/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>That would violate our python style guide: https://google.github.io/styleguide/pyguide.html#341-trailing-commas-in-sequences-of-items</p>

Reply:
interesting. thanks for letting me know